### ISSUE TITLE
Differentiate between languages in Documents List

### ISSUE THREAD

**Issue by lauradecc** [sanity-io/document-internationalization#143]
I am trying to **display the language of each document in the documents list**.

Reading the documentation I found a way to display a [list of the different singleton languages](https://github.com/sanity-io/document-internationalization/blob/main/docs/01-singleton-documents.md), but how can it be done for documents?

I tried to modify the structure based on the [Structure Builder Reference documentation](https://www.sanity.io/docs/structure-builder-reference), but I can only list the documents with `documentTypeListItem`, and I don't know how to access each document and its data. In the case of singletons, we know in advance what we want to create, while with documents we don't (for example, if we were talking about blog posts, they would be created from sanity studio).

I saw something similar to what I want to do, but [creating a custom tool](https://github.com/sanity-io/document-internationalization/blob/main/docs/04-importing-plugin-components.md#documentinternationalizationmenu). I don't quite understand how to use that documentation, but still, I need it on the desk to diff when creating/editing.

[This issue (Differentiating between languages in reference selector)](https://github.com/sanity-io/document-internationalization/issues/82) is related, as I would also like to achieve that, but I think it's not the same, and it hasn't been answered yet.

In summary, as an example, if we had two languages and with the same titles, **how could we show the language in the list of documents?**
![Captura de pantalla 2023-06-12 a las 12 27 33](https://github.com/sanity-io/document-internationalization/assets/82395407/91ab6d51-346b-4a2d-801c-586fc3624b70)

**This is the goal** (but with documents list):
![image](https://github.com/sanity-io/document-internationalization/assets/82395407/8913500f-99c3-4012-affa-eff8c7247da9)

I understand that there should be a way to do it, because the more languages, the more difficult it will be to deal with if we don't know the language (both with references and for editing, for example).

Thanks in advance!

**Comment by deanfields** [comment]
I'm quite new to Sanity CMS so apologies if this doesn't help at all, but I have just done this by putting this in the schema for the document:

``` preview: {
    select: {
      title: 'title',
      language: 'language',
    },
    prepare({ title, language }) {

      const formattedLanguage = language ? `(${language.toUpperCase()})` : null
      const titleFormat = language ? `${title} ${formattedLanguage}` : title

      return {
        title: titleFormat,
        subtitle: 'Page',
      }
    },
  },```

**Comment by lauradecc** [comment]
This was exactly what I needed! And it also solves the other issue I mentioned in the description.

Here the documentation that I found about it after reading your comment: [List Previews](https://www.sanity.io/docs/previews-list-views). It may be useful for someone else.

Thank you very much!
